instrumentation/graphql/lib/opentelemetry/instrumentation/graphql/tracers/graphql_trace.rb

```diff
@@ -15,9 +15,9 @@ module Tracers
         module GraphQLTrace # rubocop:disable Metrics/ModuleLength
           def initialize(trace_scalars: false, **_options)
             @trace_scalars = trace_scalars
-            @platform_field_key_cache = Hash.new { |h, k| h[k] = platform_field_key(k) }
-            @platform_authorized_key_cache = Hash.new { |h, k| h[k] = platform_authorized_key(k) }
-            @platform_resolve_type_key_cache = Hash.new { |h, k| h[k] = platform_resolve_type_key(k) }
+            @opentelemetry_field_key_cache = Hash.new { |h, k| h[k] = opentelemetry_field_key(k) }
```

Comment:
similarly for internal attribute names. 
```suggestion
            @_otel_field_key_cache = Hash.new { |h, k| h[k] = opentelemetry_field_key(k) }
```